src/sidebar/services/api.js

```diff
@@ -136,6 +136,12 @@ function createAPICall(
           body: data ? JSON.stringify(stripInternalProperties(data)) : null,
           headers,
           method: descriptor.method,
+        }).catch(err => {
+          // Re-throw Fetch errors such that they all "look the same" (different
+          // browsers throw different Errors on Fetch failure). This allows
+          // Fetch failures to be either handled in particular ways higher up
+          // or for them to be ignored in error reporting (see `sentry` config).
+          throw new Error(`Fetch operation error: ${err}`);
```

Comment:
I have zero attachment to this string or structure; feel free to suggest alternatives

Reply:
Looking at existing reports on Sentry, the error thrown by `fetch` doesn't have any useful information so it might not be worth capturing here. It would be useful to capture the base URL (`url` variable in this context) though. Perhaps create an `Error` subclass?